firmware/brake/kia_soul_petrol/src/main.cpp

```diff
@@ -10,21 +10,24 @@
 #include "communications.h"
 #include "debug.h"
 #include "init.h"
+#include "oscc_eeprom.h"
 #include "timers.h"
 
 
 int main( void )
 {
     init_arduino( );
 
-    init_communication_interfaces( );
-
     init_globals( );
 
     init_devices( );
 
     init_communication_interfaces( );
 
+    #ifdef RESET_CONFIG
```

Comment:
If this firmware is flashed to the device, then the EEPROM will be rewritten with each power cycle of the arduino, right?

Reply:
Only if they use `-DRESET_CONFIG=ON` during CMake configuration, which will flash our default values. Otherwise EEPROM is only written to when a configuration message is received.